src/main/scala/org/broadinstitute/hail/variant/Genotype.scala

```diff
@@ -173,6 +173,16 @@ class Genotype(private val _gt: Int,
 
   def nNonRefAlleles: Option[Int] = Genotype.nNonRefAlleles(_gt)
 
+  def oneHotGt(v: Variant): Option[Array[Int]] = {
+    gt.map { call =>
+      val gtPair = Genotype.gtPair(call)
+      val a = Array.fill(v.nAlleles)(0)
+      a(gtPair.j) += 1
+      a(gtPair.k) += 1
+      a
```

Comment:
`oneHotGt` sums to two?  I was expecting a vector over genotypes.  That's confusing.  How about `oneHotAlelles`?